scripts/stubsabot.py

```diff
@@ -435,21 +438,22 @@ async def determine_action(stub_path: Path, session: aiohttp.ClientSession) -> U
 
     diff_info = await get_diff_info(session, stub_info, pypi_info, relevant_version)
     if diff_info is not None:
-        github_repo_path, old_tag, new_tag, diff_url = diff_info
+        *_, diff_url = diff_info
```

Comment:
diff_info is already a namedtuple, so let's use attribute access in both places instead of futzing with unpacking logic.

Reply:
Didn't see it was a named tuple. Yeah that'll be much better, even than teh original code.